docs/sphinx/source/installation.rst

```diff
@@ -3,27 +3,26 @@
 Installation
 ============
 
-Installing pvlib-python is similar to installing most other scientific
-python packages. The instructions below describe how to install
-pvlib-python under a few different conditions. Most of the information
-on the `Pandas installation page
-<http://pandas.pydata.org/pandas-docs/stable/install.html>`_ is also
-applicable to pvlib-python.
+Installing pvlib-python ranges from trivial to difficult depending
+on your python experience, how you want to use pvlib, and your
+system configuration.
 
-If you have Python
-------------------
+**Do you already have Python and the NumPy and Pandas libraries?**
 
-To obtain the most recent stable release of pvlib-python, use
-`conda <http://conda.pydata.org/docs/>`_ or `pip <https://pip.pypa.io>`_::
+    If the answer to this is *No*, follow the :ref:`nopython` instructions
+    to obtain the Anaconda Python distribution before proceeding.
 
-    conda install -c pvlib pvlib
+**Do you want to use the pvlib-python as-is, or do you want to be
+able to edit the source code?**
 
-    pip install pvlib
+    If you want to use pvlib-python *as-is*, follow the simple
+    :ref:`standardrelease` instructions.
 
-If your system complains that you don't have access privileges or asks
-for a password then you're probably trying to install pvlib into your
-system's Python distribution. This is usually a bad idea and you should
-instead follow the :ref:`nopython` instructions below.
+    If you want to be able to *edit the source code*, follow the
+    :ref:`editablelibrary` instructions.
+
+Installing pvlib-python is similar to installing most scientific python
```

Comment:
There are also automated Ubuntu builds.
But since many of the ubuntu packages are quite outdated (e.g. jupyter) this may not be relevent. Once situation changes at their end, we can update the text here.